bookops_worldcat/query.py

```diff
@@ -49,12 +50,16 @@ def __init__(
         if not isinstance(prepared_request, PreparedRequest):
             raise TypeError("Invalid type for argument 'prepared_request'.")
 
+        # allow session to retry a request up to 3 times
```

Comment:
I think a better place to set up retires is `_session.WorldcatSession` class. The default number of attempts can be as given, but it would be a good strategy to give control of this element to users, i.e. instating a session should allow to pass max_retries arg. Why `WorldcatSession`? This way it will be required only once to mount the `Retry` on the session, instead of doing it in each request. If expanded to Search API these rules will apply to these sessions as well. The other arguments (`backoff_factor` & `status_forelist`) are probably OK to be constant. 


-